code/src/components/LoadingPage.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Lottie from 'lottie-react'
+import styled from 'styled-components';
+import LoadingHearts from '../lotties/love-load';
+
+export const LoadingPage = () => {
+  const defaultOptions = {
+    loop: true,
+    autoplay: true,
+    animationData: LoadingHearts,
+    rendererSettings: {
+      preserveAspectRatio: 'xMidYMid slice'
+    }
+  }
+  return (
+    <LoadingPageWrapper>
+      <Lottie animationData={LoadingHearts} options={defaultOptions} />
+    </LoadingPageWrapper>
+  )
+}
```

Comment:
The way you are handling the delay and waiting time is very cute and “gamey” – we like it a lot!